src/modules/Controller/usePointerSwipe.ts

```diff
@@ -14,7 +16,13 @@ enum Gesture {
 const SWIPE_THRESHOLD = 30;
 
 export function usePointerSwipe<T extends Element = Element>(
-  { disableSwipeNavigation }: ControllerSettings,
+  {
+    disableSwipeNavigation,
+    closeOnBackdropClick,
+    onClose,
```

Comment:
Let's add the `onClose` callback as the last argument of the function after the `onPullCancel`.